assets/styl/_modules/buttons-icon-default.twig

```diff
@@ -0,0 +1,5 @@
+<p>
+    <a href="#" class="btn-icon float-left {{ modifier_class }} mr1" data-icon="mail"><em>Call to action</em></a>
```

Comment:
At the moment, I need it for presentation purpose. Many templates currently use styles and classes that are meant to be removed when applying colette styles to all pages. 
I've created the issue #62 for that purpose.